src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+
+  margin-top: 8px;
+  margin-bottom: 8px;
+  column-gap: 17px;
+
+  &-amount {
+    display: flex;
+
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    text-align: right;
+
+    color: $main-text-color;
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  margin: 8px 0 12px;
+  gap: 4px;
+
+  &__star {
+    display: flex;
+    background-image: url(/src/images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:nth-child(-n + 4) {
+      background-image: url(/src/images/star-active.svg);
+    }
+  }
+}
```

Comment:
The `stars` class seems to be a separate block, not an element of the `card`. It might be better to move it to a separate file. Also, consider using a loop to generate the styles for the stars, it would make your code more maintainable and scalable.